pkgs/os-specific/linux/kvmtool/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "kvmtool";
```

Comment:
This should be `pname` (or include the version.)

> Specifying a name and a src is the absolute minimum Nix requires. For convenience, you can also use pname and version attributes and mkDerivation will automatically set name to "${pname}-${version}" by default. Since RFC 0035, this is preferred for packages in Nixpkgs
 — https://nixos.org/nixpkgs/manual/#sec-using-stdenv

This is for `nix-env`, where it only looks at the `name` field of derivations to distinguish between versions.